qa/L0_model_config/autofill_noplatform_success/tensorflow_graphdef/reshape_config_provided/expected

```diff
@@ -0,0 +1,58 @@
+name: "reshape_config_provided"
+platform: "tensorflow_graphdef"
```

Comment:
It is strange to provide "platform" in the config for autofill_**noplatform**_success subtests. You can remove the platform field from here and everywhere.

Ideally, you should create another directory named "autofill_success" and place these test cases over there. But I am fine with just removing platform fields from here. Specifying backend field should be sufficient.

Reply:
I might have understood something, please correct me if I'm wrong, but aren't we expecting the platform field in every expected files and only removing the platform filed from the config files?

Reply:
My bad! This was expected file. You are good!